source/playbooks/AFSBP/ssmdocs/AFSBP_RDS.13.yaml

```diff
@@ -39,8 +39,8 @@ mainSteps:
   - name: ParseInput
     action: 'aws:executeScript'
     outputs:
-      - Name: 'DbiResourceId'
-        Selector: '$.Payload.resource.Details.AwsRdsDbInstance.DbiResourceId'
+      - Name: 'DBInstanceIdentifier'
+        Selector: '$.Payload.resource.Details.AwsRdsDbInstance.DBInstanceIdentifier'
```

Comment:
Was this not working before? Or does this property have a different value?

Reply:
This property has a different value, it corresponds to the name given by the user. I'd definitely rather use the resourceId, but instance identifier is a required parameter.